tests/utils/request_utils.go

```diff
@@ -82,3 +82,8 @@ func DecodeRPC(t *testing.T, body []byte, target interface{}) error {
 	require.Nil(t, err, "respBody", string(body))
 	return nil
 }
+
+// NewEndpoint will create a new endpoint string based on hostname and port
+func NewEndpoint(hostname, port string) string {
+	return "http://" + hostname + ":" + port
+}
```

Comment:
it looks like the only hostname that's passed into this function is `utils.HOSTNAME`, perhaps we could simplify this so that it only needs the `port` and uses `utils.HOSTNAME`

Reply:
Well, you created this func before. Its not needed at all, but since you created it, I made it more generic so in case the target host differs, we can at least use it. Right now, having a func to concat 2 fixed strings + a var makes it not nice IMHO.

Reply:
please make recommended update, I agree with @noot here

Reply:
well, ok then